test/images/README.md

```diff
@@ -0,0 +1,117 @@
+# Kubernetes test images
+
+## Overview
+
+All the images found here are used in Kubernetes tests that ensure its features and functionality.
+The images are built and published as manifest lists, allowing multiarch and cross platform support.
+
+This guide will provide information on how to: make changes to images, bump their version, build the
+new images, test the changes made.
+
+
+## Prerequisites
+
+In order to build the docker test images, a Linux node is required. The node will require `make`
+and `docker (version 18.06.0 or newer)`. Manifest lists were introduced in 18.03.0, but 18.06.0
+is recommended in order to avoid certain issues.
+
+The node must be able to push the images to the desired container registry. Depending on the
+container registry, it might require a different authentication method. For dockerhub, this
+can be done by running the command:
+
+```bash
+    docker login -u your-awesome-username -p anAwesomerPassword
+```
```

Comment:
do we really need an inline docker login tutorial in every place we have images?

Reply:
agree this should go in favor of a more generic "make sure you are authenticated with the registry you're pushing to" 

in this case we're pushing to gcr.io for which this method wouldn't work, and the project has explicitly abandoned dockerhub